container/openejb-core/src/test/java/org/apache/openejb/resource/AutoConnectionTrackerTest.java

```diff
@@ -154,38 +152,29 @@ public void test() throws Exception {
 
         {
             logCapture.clear();
-            runTest(new Runnable() {
-                @Override
-                public void run() {
-                    try {
-                        bean.nonLeakyTxMethod();
-                        System.gc();
```

Comment:
I suspect that System.gc() is in there on purpose to simulate something odd. Please don't remove that.

Reply:
I restored back System.gc(); in the new commit.